config/schema/register.schema.yml

```diff
@@ -0,0 +1,12 @@
+commerce_pos.register.*:
```

Comment:
This file is misnamed, it needs to have the module name. "commerce_pos.schema.yml"